core/pkg/opencost/summaryallocation.go

```diff
@@ -151,6 +152,8 @@ func (sa *SummaryAllocation) Add(that *SummaryAllocation) error {
 	sa.RAMCost += that.RAMCost
 	sa.SharedCost += that.SharedCost
 
+	// zero out the effieciency, as it is no longer valid
```

Comment:
Is this comment still accurate? Looks like we're recalculating Efficiency after adding together the two SummaryAllocations